lib/solidus_segment/configuration.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module SolidusSegment
+  def self.configure
+    yield configuration
+  end
+
+  def self.configuration
+    @configuration ||= Configuration.new
+  end
+
+  class Configuration
+    attr_writer :segment_backend
+    attr_accessor :segment_write_key
+
+    def segment_backend
+      @segment_backend ||= if defined?(Segment::Analytics)
```

Comment:
What happens if I try to use the extension and `Segment::Analytics` is not defined?

Reply:
Please see this comment https://github.com/nebulab/solidus_segment/pull/1#discussion_r412109458